src/components/table/OcTableFiles.vue

```diff
@@ -388,6 +390,54 @@ export default {
 
       return this.$gettext("Select file")
     },
+
+    getSharedWithAvatarDescription(resource) {
+      const users = resource.sharedWith.filter(u => !u.link)
+      const links = resource.sharedWith.filter(u => !!u.link)
+      let userList
+      let linkList
+
+      if (users.length) {
+        userList = users.map(user => user.displayName).join(", ")
+      }
+
+      if (links.length) {
+        linkList = links.map(link => link.name).join(", ")
+      }
+
+      if (resource.type === "folder") {
+        if (userList && linkList) {
+          return `${this.$gettext(
+            "This folder is shared with the users"
+          )} ${userList}. ${this.$gettext("and via the links")} ${linkList}.`
+        } else if (users) {
+          return `${this.$gettext("This folder is shared with the users")} ${userList}.`
+        } else {
+          return `${this.$gettext("This folder is shared via the links")} ${linkList}.`
+        }
+      }
+
+      if (userList && linkList) {
+        return `${this.$gettext("This file is shared with the users")} ${userList}. ${this.$gettext(
+          "and via the links"
+        )} ${linkList}.`
+      } else if (users) {
+        return `${this.$gettext("This file is shared with the users")} ${userList}.`
+      } else {
+        return `${this.$gettext("This file is shared via the links")} ${linkList}.`
+      }
+    },
+
+    getOwnerAvatarDescription(resource) {
+      let description
+      if (resource.type === "folder") {
+        description = this.$gettext("This folder is owned by")
```

Comment:
Please use `$gettextInterpolate` wherever you inject/concatenate additional variables to a string.

Reply:
Reason is simple: in other languages the name might not be at the end of the string but somewhere in the middle, depending on the respective grammar of the language. Placing the variable at the correct location is a responsibility of the translation, not for the code.